consensus/progpow/poem.go

```diff
@@ -288,3 +286,8 @@ func (progpow *Progpow) CheckIfValidWorkShare(workShare *types.WorkObjectHeader)
 	}
 	return new(big.Int).SetBytes(powHash.Bytes()).Cmp(workShareMintarget) <= 0
 }
+
+func (progpow *Progpow) DiffToBigBits(header *types.WorkObject) *big.Int {
+	target := new(big.Int).Div(common.Big2e256, header.Difficulty())
+	return progpow.IntrinsicLogS(common.BytesToHash(target.Bytes()))
+}
```

Comment:
Any changes from blake3 should also be applied here

Reply:
ok